src/components/settings/ProfileField.tsx

```
@@ -0,0 +1,95 @@
+'use client';
+
+import Image from 'next/image';
+import { faPenToSquare } from '@fortawesome/free-regular-svg-icons';
+import { useEffect, useState } from 'react';
+import FormTitle from './FormTitle';
+import Icon from '@/components/common/icon/Icon';
+
+interface Props {
+  url: string;
+  onChange: (file?: File) => void;
```

ESLint

[error] 'file' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)